aptos-move/block-executor/src/captured_reads.rs

```diff
@@ -462,6 +521,31 @@ pub(crate) struct CapturedReads<T: Transaction, K, DC, VC, S> {
     /// Set if the invariant on CapturedReads intended use is violated. Leads to an alert
     /// and sequential execution fallback.
     incorrect_use: bool,
+
+    data_read_comparator: DataReadComparator,
+}
+
+impl<T: Transaction, K, DC, VC, S> Default for CapturedReads<T, K, DC, VC, S> {
+    fn default() -> Self {
+        Self::new(None)
+    }
+}
+
+impl<T: Transaction, K, DC, VC, S> CapturedReads<T, K, DC, VC, S> {
+    #[allow(deprecated)]
```

Comment:
What is deprecated?

Reply:
I had seen it was used to add to summaries and thought it was for replay or smt. But now checking it is seems it is never accessed otherwise so should be always empty. Deleting in the coming refactor / post-addressing comments PR.